src/main/java/com/hazelcast/kubernetes/DefaultKubernetesClient.java

```diff
@@ -296,21 +295,16 @@ private static SSLSocketFactory buildSslSocketFactory() {
     /**
      * Generates CA Certificate from the default CA Cert file (which always exists in the Kubernetes POD).
```

Comment:
"... from _either_ the default CA... _or from the externally provided <code>ca-certificate</code> property_."